src/model/core/co2.jl

```diff
@@ -0,0 +1,122 @@
+@doc raw""" 
+    co2!(EP::Model, inputs::Dict)
+
+This function creates expression to account for CO2 emissions and captured and sequestrated 
+CO2 from thermal generators. It also has the capability to model the negative CO2 emissions 
+from bioenergy with carbon capture and storage. 
+
+***** Expressions *****
+
+For thermal generators use fuels that contain CO2 content (e.g., coal, natural gas, and biomass), 
+the CO2 emissions are a function of fuel consumption, CO2 capture rate, and whether the feedstock 
+is biomass. 
+Biomass (e.g., wastes or agriculture resides) derived energy is typically considered to be 
+carbon-neutral because the carbon in the biomass is originated from the atmosphere. 
+When bioenergy is coupled with carbon capture and storage (CCS), it creates negative emissions.
+If users want to represet delicated biomass, then in Generators_data.csv, it requires a column
+name called "BIOMASS" (boolean, 1 or 0), which represents if a generator $y$ uses biomass or not.
+The CO2 emissions from a generator should be zero without CCS and negative with CCS.
+The CO2 emissions from the generator $y$ at time $t$ (commited), is determined by total fuel 
+consumption (MMBTU) multiplied by the CO2 content of the fuel (t CO2/MMBTU), 
+then times (1 - BIOMASS [0 or 1] - CO2 capture rate [a fraction, between 0 - 1]). 
+The CO2 capture rate during the steady-state and startup event could be differernt 
```

Comment:
The CO2 capture rates could be different during steady-state operation and during startup events